piqueserver/core_commands/info.py

```diff
@@ -13,21 +14,16 @@ def streak(connection):
 
 
 @command()
-def ping(connection, value=None):
+@target_player
+def ping(connection, player):
     """
     Tell your current ping (time for your actions to be received by the server)
     /ping
     """
-    if value is None:
-        if connection not in connection.protocol.players.values():
-            raise ValueError()
-        player = connection
-    else:
-        player = get_player(connection.protocol, value)
     ping = player.latency
-    if value is None:
-        return 'Your ping is %s ms. Lower ping is better!' % ping
-    return "%s's ping is %s ms" % (player.name, ping)
+    if connection == player:
```

Comment:
you really want to check using `is` here, as you are checking identity, not equality